iXBRLViewerPlugin/viewer/src/html/inspector.html

```diff
@@ -86,6 +86,15 @@ <h3 data-i18n="inspector.filter">Filter</h3>
             <option value="summationOrContributor" data-i18n="inspector.summationOrContributor">Summation or Contributor</option>
           </select>
 
+          <div class="control-label" data-i18n="inspector.dimensions">
+              Dimension Type
+          </div>
+          <select id="search-filter-dimension-type">
+              <option value="*" data-i18n="inspector.noFilter">-</option>
+              <option value="explicit" data-i18n="inspector.explicitDimension">Explicit Dimensions</option>
+              <option value="typed" data-i18n="inspector.typedDimensions">Typed Dimensions</option>
```

Comment:
Would there be value in having "No Dimensions" be an option?

Reply:
Yeah that's probably reasonable

Reply:
Going to proceed as is and circle back with an issue to improve the filter

Reply:
https://github.com/Workiva/ixbrl-viewer/issues/431